web/src/presentation/App.js

```diff
@@ -3,16 +3,18 @@ import PropTypes from 'prop-types';
 import { Router } from './Router';
 import { Provider } from 'react-redux';
 import './App.css';
+import { identifyUser } from '../data/log-rocket';
```

Comment:
Perceba que nossa camada de apresentação esta acessando a camada de dados diretamente. Isso não é bom, vou te dizer o porque:
(Este exemplo não vai fazer muito sentido porque nossa aplicação ainda é pequena, mas as complexidades vao aumentando a medida que vamos adicionando coisas).

Suponha que no futuro a gente mude esse listener para não atualizar o estado do usuario caso o usuario venha direto do login. e.g, atualmente se um usuario logar nos vamos alterar o user duas vezes.
A chamada do `identifyUser` está ligada somente ao listener e não a regra de negocio em si. 
Agora nos temos uma regra de negocio:
- Quando o usuário logar, precisamos notificar para a nossa ferramenta de monitoramento o usuário em questão.

Um simples control c + control v resolve isso, porém o que acontece se colocassemos outros pontos de entrada (login)? Teriamos que copiar e colar daqui pra lá. Copy-and-Paste approach raramente é a melhor solução. 
O ideal é ter um ponto de entrada aonde podemos garantir que nossas regras de négocio sempre funcionem.